tests/resources/functions/flutter/latest/lib/main.dart

```diff
@@ -0,0 +1,125 @@
+import 'package:flutter/material.dart';
+
+void main() {
+  runApp(const MyApp());
+}
+
+class MyApp extends StatelessWidget {
+  const MyApp({super.key});
+
+  // This widget is the root of your application.
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      title: 'Flutter Demo',
+      theme: ThemeData(
+        // This is the theme of your application.
+        //
+        // TRY THIS: Try running your application with "flutter run". You'll see
```

Comment:
Do we need all of these comments in this file?

Reply:
Nice catch, it was auto-generated by Flutter. Simplified Flutter test files.